src/parser.rs

```diff
@@ -599,18 +598,19 @@ impl Library {
                                     "name" => field_name = attr.value.clone(),
                                     _ => {},
                                 }
+                                break;
```

Comment:
Ah, I was blind and saw the break one layer outside for the other loop. Sure, makes sense